isubit/iastate_theme
# Development

This pull request does 3 things

1. Improves base typography, overriding Bootstrap 4
2. We will now host academicons because their remote hosting is going away.
3. There is now a theme setting for showing/hiding the gold border on the red header

**To test:**
1. Copy in some real content from our faculty sites and observe that the headings have a generally pleaseing size and spacing.
2. Add researchgate and googlescholar links in social media areas, like the footer.
3. Go to Appearance theme settings and turn the gold border toggle on and off.
